pilz_teleoperation/config/keyboard_binding.yaml

```diff
@@ -60,3 +61,4 @@ Description: |-
   
   numpad : move inside the selected plane
   , : change plane
+  
```

Comment:
why removing it?

Reply:
Newline at eof is there, see https://github.com/PilzDE/pilz_teach/blob/usability_test_additions/pilz_teleoperation/config/keyboard_binding.yaml

Reply:
Newline at eof is there, see https://github.com/PilzDE/pilz_teach/blob/usability_test_additions/pilz_teleoperation/config/keyboard_binding.yaml